### ISSUE TITLE
distill.py   loss = nan

### ISSUE THREAD

**Issue by yangyangtiaoguo** [GeorgeCazenavette/mtt-distillation#6]
Hello, author. Thank you for your work.！
Running distill During py, loss is always Nan. What parameters do the author suggest to adjust? Or did I ignore what caused the error？
In addition: I use my own dataset. The experimental settings and dataset settings are shown in the figure below.
![image](https://user-images.githubusercontent.com/14866214/161431164-387cf9a0-4c1b-4309-b9a4-2c40aab899aa.png)
![image](https://user-images.githubusercontent.com/14866214/161431193-02a775c6-5f72-4bdc-8367-c3948ccefb3e.png)

**Comment by GeorgeCazenavette** [comment]
Is it NaN loss from the very first iteration?

Is this only happening when you use your own dataset?

Did your experts seem to train okay? If so, around what epoch did they seem to converge?

What model are you using? We used ConvNetD4 for 64x64 images.
